tests/conftest.py

```diff
@@ -222,6 +224,19 @@ def pytest_configure(config):
     called after command line options have been parsed
     and all plugins and initial conftest files been loaded.
     """
+    # try:
```

Comment:
Why are we commenting here and not removing?

Reply:
Because it will need to get added back, once I have time to solve the issue preventing the use of this specific code in salt-factories.